From a.beregalov@gmail.com  Wed Feb  4 14:56:37 2009
From: Alexander Beregalov <a.beregalov@gmail.com>
Date: Wed, 4 Feb 2009 15:38:37 +0300
Subject: Staging: benet: convert netif_rx_* to napi_*
To: Sathya Perla <sathyap@serverengines.com>, Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <20090204123837.GA12987@orion>
Content-Disposition: inline


  *netif_rx_* functions were removed by 288379f ("net: Remove redundant NAPI
  functions")

Signed-off-by: Alexander Beregalov <a.beregalov@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/benet/be_int.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/benet/be_int.c
+++ b/drivers/staging/benet/be_int.c
@@ -717,7 +717,7 @@ int be_poll(struct napi_struct *napi, in
 
 	/* All consumed */
 	if (work_done < budget) {
-		netif_rx_complete(napi);
+		napi_complete(napi);
 		/* enable intr */
 		be_notify_cmpl(pnob, work_done, pnob->rx_cq_id, 1);
 	} else {
@@ -752,7 +752,7 @@ static inline u32 process_events(struct 
 		rid = AMAP_GET_BITS_PTR(EQ_ENTRY, ResourceID, eqp);
 		if (rid == pnob->rx_cq_id) {
 			adapter->be_stat.bes_rx_events++;
-			netif_rx_schedule(&pnob->napi);
+			napi_schedule(&pnob->napi);
 		} else if (rid == pnob->tx_cq_id) {
 			process_nic_tx_completions(pnob);
 		} else if (rid == pnob->mcc_cq_id) {
